services/web/server/src/simcore_service_webserver/users_handlers.py

```diff
@@ -37,6 +38,9 @@ async def _query_db(uid, engine):
     )
     parts = row["name"].split(".") + [""]
 
+    session = await get_session(request)
+    session["user_email"] = row["email"]
+
```

Comment:
Why do you need to set this value in two handlers (user_handers and login handler)?? Shoudn't be enough only upon login?

Reply:
This is because once it deploys, the users already logged in, that don't have to go through the login endpoint, won't get this inside their sessions unless they logout and login again

Reply:
basically is to be able to preserve session between deployment updates and have everything working

Reply:
OK. Could you please just add a note above the code. thx

Reply:
done